pom.xml

```diff
@@ -105,6 +105,7 @@
     <error.prone.annotations.version>2.1.3</error.prone.annotations.version>
     <jsr305.version>3.0.2</jsr305.version>
     <commons.lang3.version>3.8.1</commons.lang3.version>
+    <google.turbine.version>v20220503-SNAPSHOT</google.turbine.version>
```

Comment:
I'll finish this release so you can skip SNAPSHOT here, since it looks like we're solid, just some future plans to stop requiring a zip right?